### PR TITLE
test(e2e-tests): refactored e2e - mainly wording and style of feature

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
UCD-1277

## Description of change
Refactored wording and style of feature files for Your Details in line with feedback.

### CODE REVIEW

File: e2e-tests/cypress/integration/common/appellant-submission-your-details.js [hunk 1]
```diff
@@ -120,6 +149,35 @@ Then("the appeal's Your Details task is completed with {string} and {string}", (
...
+      break;
+    case 'name with prohibited characters':
+      cy.confirmDetailsWasRejected('Name must only include letters a to z, hyphens, spaces and apostrophes');
+      break;
+    default:
+      throw new Error(`test fails here because it could not find the reason [${reason}] in the list of cases`)
+  }
+});
+
+Then('email {string} is invalid because {string}', (nameOrEmail, reason) => {
```
Comment (frank-dodion) [inline e2e-tests/cypress/integration/common/appellant-submission-your-details.js:168]: I will replace nameOrEmail with email on next PR.

**Review by GoEagles999 — APPROVED** [review]

File: e2e-tests/cypress/integration/appellant-submission-your-details-v2-applicant-name.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@smoketest
```
Comment (a6software) [inline e2e-tests/cypress/integration/appellant-submission-your-details-v2-applicant-name.feature:1]: what does `v1` and `v2` in the file names represent ? 

It seems like 'versions' is the obvious answer, but I've never seen this done before so am curious.

**Review by Fenisay — APPROVED** [review]

**Review by a6software — APPROVED** [review]
really nice 👍

File: e2e-tests/cypress/integration/appellant-submission-your-details-v2-applicant-name.feature (hunk 2 above)
Comment (frank-dodion) [reply e2e-tests/cypress/integration/appellant-submission-your-details-v2-applicant-name.feature:1]: I have used it to make clear version 1 of how we did feature files from the latest v2. I did not want to remove v1 just yet but I think we will do so in the future.

**Pull request merged**